src/container/srv_container.c

```diff
@@ -2126,6 +2128,7 @@ cont_lookup(struct rdb_tx *tx, const struct cont_svc *svc, const uuid_t uuid, st
 err_p:
 	D_FREE(p);
 err:
+	DL_ERROR(rc, "cont " DF_UUID ", lookup failed.\n", DP_UUID(uuid));
```

Comment:
Same comment as above. Also, if these errors are kept, probably don't need "cont " in the output since the function name already has that.